assembly/src/assembler/instruction/crypto_ops.rs

```diff
@@ -191,3 +178,22 @@ fn read_mtree_node(span: &mut SpanBuilder) {
     // MRUPDATE: [V_old, d, i, R, V_new, ...]
     span.push_op_many(Read, 4);
 }
+
+/// Update a node in the merkle tree. The `copy` flag will be passed as argument of the `MrUpdate`
+/// operation.
+fn update_mtree(span: &mut SpanBuilder, copy: bool) {
+    #[rustfmt::skip]
+    span.push_ops([
+        // Update the Merkle tree with the new value without copying the old tree. This replaces the
+        // old node value with the computed new Merkle root.
+        // => [R_new, d, i, R, V_new, ...]
+        MrUpdate(copy),
+
+        // move d, i back to the top of the stack and are dropped since they are
+        // no longer needed => [R_new, R, V_new, ...]
+        MovUp4,
+        Drop,
+        MovUp4,
+        Drop,
```

Comment:
minor nit: should these be listed in one row?

Reply:
Fixed